core/pom.xml

```diff
@@ -200,6 +201,13 @@
             <artifactId>log4j-1.2-api</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <!--    paimon    -->
+        <dependency>
```

Comment:
Can you manage the dependency in the `DependencyManagement` blocker in parent pom?